### PR TITLE
Article about SnabbWall's L7Spy

### PR DESCRIPTION
**Pull request by aperezdc**
This is still WIP, there's a couple of sections missing. Posting the PR already so @dpino can proofread what is already here :wink:

### CODE REVIEW

**Comment by aperezdc** [comment]
@dpino: PTAL.

File: 2016/snabbwalls-l7spy-analyzer.markdown [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+releases. I noticed how it might end up being difficult to
+[rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) SnabbWall on
+top of the latest release, and switched to the following scheme:
+
+* The branch containing the most up-to-date development code is always in the
+  `snabbwall` branch.
+* Whenever a new Snabb version is released, changes are merged from the
+  upstream repository using its corresponding release tag.
+* When a SnabbWall release is done, it will be based on the most
+  recently-merged Snabb release. A tag release tag will be created for it.
```
Comment (dpino) [inline 2016/snabbwalls-l7spy-analyzer.markdown]: Maybe should be "A release tag will be created for it"?

File: 2016/snabbwalls-l7spy-analyzer.markdown [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+different ways, which should allow for painless integration. The application
+has two bidirectional endpoints called *south* and *north*:
+
+<figure style="text-align:center">
+  ![](https://perezdecastro.org/2016/app-l7spy.png)
+  <figcaption>The L7Spy application.</figcaption>
+</figure>
+
+Packets entering one of the ends are forwarded unmodified to the other end,
+in both directions. Packets are scanned as thet flow through the application.
```
Comment (dpino) [inline 2016/snabbwalls-l7spy-analyzer.markdown]: s/thet/they/

File: 2016/snabbwalls-l7spy-analyzer.markdown (hunk 1 above)
Comment (aperezdc) [inline 2016/snabbwalls-l7spy-analyzer.markdown]: Yes, your wording seems easier on the eyes, I'll change that.

File: 2016/snabbwalls-l7spy-analyzer.markdown (hunk 2 above)
Comment (aperezdc) [inline 2016/snabbwalls-l7spy-analyzer.markdown]: Good catch, fixing.

**Comment by aperezdc** [comment]
Merged with the fixes suggested by @dpino applied. Thanks for reviewing!

**Pull request merged**